src/public/index.html

```diff
@@ -16,107 +16,129 @@
+    <div id="homeContainer" class="card">
+        <div class="content">
+            <div class="titleCard">Choose a file to import</div>
+            <div class=uploadContainer>
+                <label for="importFile" class="fileUpload">Browse file</label>
+                <input type="file" id="importFile" name="importFile"><br>
+                <p id="fileName" class="fileName "></p>
+                <div id="fileError"></div>
+            </div>
+        </div>
+        <button id="readButton" class="primaryButton">Upload</button>
+        <button id="showExampleButton" class="link-button">
+            How to structure my file
+        </button>
     </div>
-    <div id="cardsContainer">
-        <div id="homeContainer">
-            <div class="content">
-                <div class="titleCard">Choose a file to import</div>
-                <div class=uploadContainer>
-                    <label for="importFile" class="fileUpload">Browse file</label>
-                    <input type="file" id="importFile" name="importFile"><br>
-                    <p id="fileName" class="fileName "> </p>
-                    <div id="fileError"></div>
-                </div>
+    <div id="fileStructureExampleContainer" class="card">
+        <div class="content">
+            <div class="titleCard">Anatomy of a CSV File</div>
+            <div class="secondary-title">Understand the structure of a node and an edge file</div>
+            <div class="tab">
+                <button id="showNodeExampleButton" class="link-button">
+                    Nodes
+                </button>
+                <button id="showEdgeExampleButton" class="link-button">
+                    Edges
+                </button>
+            </div>
+            <div class="example-image">
+                <img id="exampleImage" src="./assets/img/node.png" alt="CSV file structure example" width="651" height="413">
+            </div>
+            <div class="container_flex-end">
+                <button id="exitExampleButton" class="primaryButton">Got it</button>
             </div>
-            <button id="readButton" class="primaryButton">Upload</button>
         </div>
-        <div id="pickEntityContainer">
-            <div class="content">
-                <div class="titleCard">Which entity does the file contain?</div>
-                <div class="radioButtons">
-                    <div>
-                        <input type="radio" name="entities" id="nodes" value="nodes">
-                        <label for="nodes">Nodes</label>
-                    </div>
-                    <div>
-                        <input type="radio" name="entities" id="edges" value="edges">
-                        <label for="edges">Edges</label>
-                    </div>
+    </div>
+
+    <div id="pickEntityContainer" class="card">
+        <div class="content">
+            <div class="titleCard">Which entity does the file contain?</div>
+            <div class="radioButtons">
```

Comment:
You shoudl use a radio group here I think.

Reply:
what is a radio group ? I think we just needs to give the input the same "name" value